packaging/darwin/Distribution.in

```diff
@@ -48,11 +48,11 @@ function installCheck() {
         <line choice="crc"/>
         <line choice="crc-tray"/>
     </choices-outline>
-    <choice id="crc" title="crc">
+    <choice id="crc" title="Red Hat OpenShift Local" enabled="false" selected="true" visible="true">
         <pkg-ref id="crc.pkg"/>
     </choice>
     <pkg-ref id="crc.pkg" auth="Root">crc.pkg</pkg-ref>
-    <choice id="crc-tray" title="crc-tray">
+    <choice id="crc-tray" title="System Tray for Red Hat OpenShift Local" enabled="true" selected="true" visible="true">
```

Comment:
As part of `crc-tray` shouldn't we make `selected="false"` so that it is not selected by default when user start the installer?